src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -245,10 +273,10 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
                 case AdyenPaymentMethodType.ACH:
                 case AdyenPaymentMethodType.Bancontact:
                     paymentComponent = adyenClient.create(paymentMethod.method, {
-                            ...adyenv2.options,
-                            onChange: componentState => this._updateComponentState(componentState),
-                        }
-                    );
+                        ...adyenv2.options,
+                        onChange: componentState => this._updateComponentState(componentState),
+                        data: this._mapDataFromStoreState(),
```

Comment:
a `mapper` function usually takes one param and converts into something else.
Should this be `this._getAdyenBillingDetails()` ?